example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -252,7 +252,7 @@ describe('The Ingest Granule failure workflow', () => {
         const isSchemaValidationError = (error.Error === 'CumulusMessageAdapterExecutionError') &&
           error.Cause.includes('jsonschema.exceptions.ValidationError');
         const isPostgresWriteError = error.Error.includes('Failed writing files to PostgreSQL') &&
-          error.Cause.includes('null value in column "bucket" violates not-null constraint');
+          error.Cause.includes('null value in column "bucket" of relation "files" violates not-null constraint');
```

Comment:
This change was needed due to the enginerupdate

Reply:
Yeah, probably shouldn't make assertions against the error.Cause in this case if that's a description that might change but not a huge deal.